application/app/models/dataverse/create_dataset_response.rb

```diff
@@ -0,0 +1,13 @@
+module Dataverse
+  class CreateDatasetResponse
+    attr_reader :status, :id, :persistent_id, :message
+
+    def initialize(json)
+      parsed = JSON.parse(json, symbolize_names: true)
```

Comment:
I personally think that the parsing of JSON should be in the client and that the PORO should take a hash.
I think parsing JSON in the constructor is done in other responses. I think we should revisit and refactor (in other PR)

What do you think?

Reply:
I don't mind both ways, but if you think that the responsibility of parsing JSON should be outside of the response classes, we can refactor in another PR.

Reply:
I think so - lets refactor in another PR